processing/src/main/java/org/apache/carbondata/processing/loading/sort/unsafe/UnsafeSortDataRows.java

```diff
@@ -243,8 +244,8 @@ public void addRow(Object[] row) throws CarbonSortKeyAndGroupByException {
       }
       rowPage.addRow(row, reUsableByteArrayDataOutputStream.get());
     } catch (Exception e) {
-      if (e.getMessage().contains("cannot handle this row. create new page"))
-      {
+      if (e.getMessage().contains("cannot handle this row. create new page")) {
+        LOGGER.info("cannot handle this row. create new page");
```

Comment:
For each row, it will check sort buffer size.
Better to not log this info.

Reply:
done